src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -36,7 +36,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <MessagePackRoot>$(RepoRoot)src\submodules\MessagePack-CSharp\src\MessagePack\</MessagePackRoot>
+    <MessagePackRoot>$(RepoRoot)src\submodules\MessagePack-CSharp\src\MessagePack.UnityClient\Assets\Scripts\MessagePack\</MessagePackRoot>
```

Comment:
UnityClient??

I'm out of the loop with this one. No doubt this is all deliberate but it does look a bit odd!

Reply:
Yes, this is correct. Unity doesn't use project files, so all files for unity must be in a source directory that contains *only* unity sources. Since csproj is more flexible, our regular csproj 'reaches in' to the unity directory to get the common source files.

Reply:
FYI - this file - https://github.com/neuecc/MessagePack-CSharp/blob/master/sandbox/MessagePack.Internal/MessagePack.Internal.csproj mirrors what gets built in this project. I should add a comment to this effect for future self